use_cases/eluc/prescriptors/nsga2/nsga2_utils.py

```diff
@@ -64,7 +64,7 @@ def calculate_crowding_distance(front):
     n_objectives = len(front[0].metrics)
     distances = [0 for _ in range(len(front))]
     for m in range(n_objectives):
-        front.sort(key=lambda c: c.metrics[m])
+        front.sort(key=lambda cand: cand.metrics[m])
```

Comment:
Changed all of these to make it more clear c is a candidate

Reply:
you could go as far as calling `cand` `candidate`. That will more more readable and we can afford the extra letters